source/renderer/app/components/wallet/tokens/wallet-token-picker/helpers.js

```diff
@@ -0,0 +1,154 @@
+// @flow
+import { messages } from './WalletTokenPicker.messages';
+import {
+  MAX_TOKENS,
+  ScrollPositionEnum,
+  FilterSelectOptionsEnum,
+  ToggleModeEnum,
+} from './const';
+import type { Intl } from '../../../../types/i18nTypes';
+import type {
+  Assets,
+  BooleanMap,
+  ClearSelection,
+  FilterAssets,
+  ScrollPosition,
+  GetMaxTokensIdMap,
+  GetTokenCounterText,
+  GetCurrentCheckedIds,
+  GetToggleAllMode,
+  GetEnabledAssetIds,
+} from './types';
+
+const isScrollAtTop = (element: HTMLElement) => element.scrollTop === 0;
```

Comment:
we could move these next 3 methods into the scroll position hook, what do you think?

Reply:
My intention was to separate pure/non lifecycle related function into helper file and statefull/lifecycle functions in the hooks. What do you think?